src/routes/index.js

```diff
@@ -23,7 +23,13 @@ export default function getRoutes(renderRequestMap, apiProxyFn$ = apiProxy$) {
 		method: 'GET',
 		path: '/_ah/{param*}',
 		config: { auth: false },
-		handler: (request, reply) => reply('OK'),
+		handler: (request, reply) => {
+			if (request.params.param == !'error') {
```

Comment:
looks like a typo here - I think it should be `request.params.param !== 'error'`